### ISSUE TITLE
Read out caption on (and only on) chart landing

### ISSUE THREAD

**Issue by simonvarey** [fizzstudio/ParaCharts#973]
**Is your feature request related to a problem? Please describe.**
The caption (and therefore the whole chart summary) should be read out to screen reader users, but not every time the caption is visible.

**Describe the solution you'd like**
Read out caption on (and only on) chart landing, followed by the orientation sentences in the nav bar

**Additional context**
May depend on making the nav bar visible on initial chart landing (see #972)
